themes/bootstrap3/templates/layout/layout.phtml

```diff
@@ -118,6 +123,11 @@ VuFind.defaultSearchBackend = '{$dsb}';
 VuFind.addTranslations({$translations});
 JS;
       $this->headScript()->appendScript($setupJS);
+
+      $account = $this->auth()->getManager();
```

Comment:
Is it necessary to load this last? If not, maybe we should move the translation strings specific to this feature into the conditional block below, similar to how the cart is handled.